test/e2e/cluster_status_test.go

```diff
@@ -17,18 +17,86 @@ package e2e
 import (
 	"fmt"
 	"os"
+	"strings"
 	"testing"
 	"time"
 
+	"github.com/coreos/etcd-operator/pkg/spec"
 	"github.com/coreos/etcd-operator/pkg/util/k8sutil"
 	"github.com/coreos/etcd-operator/pkg/util/retryutil"
 	"github.com/coreos/etcd-operator/test/e2e/framework"
+
+	"k8s.io/client-go/pkg/api/unversioned"
+	"k8s.io/client-go/pkg/api/v1"
 )
 
-func TestBackupStatus(t *testing.T) {
+func TestClusterStatus(t *testing.T) {
```

Comment:
let us separate this two test. i do not feel they belong to one test group.

back status tests if backup status reports correctly.

testFailedPhase tests if controller reacts correctly when one of the cluster is failed.